src/target/riscv/riscv.h

```diff
@@ -370,8 +373,14 @@ int riscv_current_hartid(const struct target *target);
  * consecutive and start with mhartid=0. */
 unsigned int riscv_count_harts(struct target *target);
 
-/** Set register, updating the cache. */
+/** Set the register value. For cacheable registers, only the cache is updated
+ *  (write-back mode).
+ */
```

Comment:
Detail - style of the doxygen comments:

```suggestion
/** 
 * Set the register value. For cacheable registers, only the cache is updated
 * (write-back mode).
 */
```

Reply:
Addressed.